verify/runners.py

```diff
@@ -108,3 +110,58 @@ def __call__(self, *assertions):
                 assertion = verify.Predicate(assertion)
             assertion(self.value)
         return self
+
+
+ensure = expect
+
+
+def _find_assertion_class(name):
+    try:
+        return getattr(verify, name)
+    except AttributeError:
+        pass
+
+    name_formatters = [
+        _class_format,
+        _to_be_prefix,
+        _is_prefix,
+        _reserved_names,
+    ]
+
+    for format_name in name_formatters:
+        new_name = format_name(name)
+        if new_name is None:
+            continue
+
+        try:
+            return getattr(verify, new_name)
+        except AttributeError:
+            pass
+
+    raise AttributeError(('"{0}" is not a valid assertion method'
+                          .format(name)))
+
+
+def _class_format(name):
+    new_name = [part.capitalize() for part in name.split('_')]
+    return ''.join(new_name)
+
+
+def _to_be_prefix(name):
+    return _prefixed_name(name, 'to_be_')
+
+
+def _is_prefix(name):
+    return _prefixed_name(name, 'is_')
+
+
+def _prefixed_name(name, prefix):
+    if re.match(prefix, name):
+        return _class_format(name[len(prefix):])
+
+
+def _reserved_names(name):
+    if name == 'does':
```

Comment:
I wonder if a better name for these might be `to_pass`/`passes` and `to_fail`/`fails` for `Predicate` and `Not` respectively?